client/extensions/woocommerce/state/sites/shipping-zone-locations/selectors.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import { get, isObject } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { LOADING } from 'woocommerce/state/constants';
+
+const getRawShippingZoneLocations = ( state, siteId = getSelectedSiteId( state ) ) => {
```

Comment:
Right now it's just for local use, but when the UI selectors for this are implemented, the `getRawShippingZoneLocations` will be `export`ed.

What name would you suggest? This function returns "the list of locations for this shipping zone, as returned by the server/API, without local edits". `getRaw...` or `getApi` are the best names that came to my mind :)